src/Seq.App.EmailPlus/EmailApp.cs

```diff
@@ -17,11 +19,13 @@ namespace Seq.App.EmailPlus
 
     [SeqApp("Email+",
         Description = "Uses a Handlebars template to send events as SMTP email.")]
-    public class EmailApp : SeqApp, ISubscribeTo<LogEventData>
+    public class EmailApp : SeqApp, ISubscribeToAsync<LogEventData>
     {
         readonly IMailGateway _mailGateway = new DirectMailGateway();
         readonly ConcurrentDictionary<uint, DateTime> _lastSeen = new ConcurrentDictionary<uint, DateTime>();
         readonly Lazy<Template> _bodyTemplate, _subjectTemplate, _toAddressesTemplate;
+        private SmtpClient _client = new SmtpClient();
+        private SmtpOptions _options = new SmtpOptions();
```

Comment:
Missed these - should be `readonly`, no explicit `private`

Reply:
Done - no need for OnAttached now.